openfisca_france/tests/formulas/irpp.yaml

```diff
@@ -952,3 +952,20 @@
     f4ba: 150000
   output_variables:
     irpp: -48036
+- name: "IRPP - Célibataire ayant des 	Revenu net imposable de 16000 € (teste la décote)"
+  period: 2014
+  absolute_error_margin: 0.9 #TODO : check why the error margin is so large, is there bottom round made by dgfip ?
+  input_variables:
+    salaire_imposable: 16000
+  output_variables:
+    irpp: -183
+    decote: 476
+#TODO : check where the 24.5 difference in IRPP comes from.
```

Comment:
@cbenz  thank you for your answer.
>Tests must be valid in order to be merged into next, but since you are on your branch you can leave invalid tests.

@benjello  @cbenz, ok since those not passing test are new test that I added and do not break old test I taught that it was the best way to do. It also gives an incentive to hide old errors to the guy that do the pull request if one is entitled to solve any error he finds. 

>Once you fix them, the pull-request will launch Travis tests again automatically. Did you know that >if you push new commits in the pull-request branch (ie decote_2015), the pull-request updates >itself?

Yes I know I've done that 4 times only for this pull request...

>For me these invalid tests an indicator that I must not merge the pull-request :)

For current PR we've found with @benjello  the problem with my commented test (old parameters), and I also added an other test for couples. Except for style (pep8) issue or mistake found I will not push again for this new PR :-) .